### ISSUE TITLE
Update the content for our project examples

### ISSUE THREAD

**Issue by Nikomus** [neontribe/www#490]
**What needs doing?**

**Plan**
- [ ] Step one to achieve this.
- [ ] Step two to achieve this.

**Comment by katquatermass** [comment]
https://docs.google.com/document/d/10jGJjFioQIGeWEsZmAoSfzUhHismU_44RwczYqyIdS0/edit?usp=sharing   Example drafts

**Comment by katquatermass** [comment]
Believe there is enough appetite to go ahead with the new content structure.

**Comment by harryharrold** [comment]
agreed - well, at least I know I have appetite, and no-one seems *not* to.
